docs/src/getting_started/Android/android.md

```diff
@@ -16,9 +16,13 @@ We want to make setting up Android Studio to work with Crux really easy. As time
 The first thing we need to do is create a new Android app in Android Studio.
 
 Open Android Studio and create a new project, for "Phone and Tablet", of type
-"Empty Compose Activity (Material3)". In this walk-through, we'll call it
-"Counter", use a minimum SDK of API 34, and save it in a directory called
-`Android`.
+"Empty Activity". In this walk-through, we'll call it "Counter"
+
+* "Name": `Counter`
+* "Package name": `com.example.simple_counter`
+* "Save Location": a directory called `Android` at the root of our monorepo
+* "Minimum SDK" `API 34`
+* "Build configuration language": `Groovy DSL (build.gradle)`
```

Comment:
For me Android Studio was defaulting to `build.gradle.kts` build files, which ofc won't then work with the copy-pastes later.